src/test/unit/org/example/learningsystem/common/DestinationUtils.java

```diff
@@ -0,0 +1,17 @@
+package org.example.learningsystem.common;
+
+import org.example.learningsystem.btp.destinationservice.dto.MailDestinationDto;
+
+public class DestinationUtils {
```

Comment:
this class can be moved under `destinationservice` package, as it related to this feature (even it's used by another package)